content/en/developers/faq/legacy-openmetrics.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Custom Legacy OpenMetrics Check
+kind: documentation
+further_reading:
+- link: "/agent/prometheus/"
+  tag: "Documentation"
+  text: "Configuring an OpenMetrics Check"
+- link: "/developers/agent_checks/"
+  tag: "Documentation"
+  text: "Write a Custom Check"
+- link: "/developers/integrations/"
+  tag: "Documentation"
+  text: "Create a new Integration"
+aliases:
+  - /developers/openmetrics/
+  - /developers/prometheus/
```

Comment:
Do you intend to redirect users to this new page instead of the one above?

Reply:
I wasn't sure what the alias for so I left it in the copy+paste. If i only want to link users to this page from prometheus.md, I can delete these two lines?